### PR TITLE
created an array of comments and tested for errors

### PR DESCRIPTION
**Pull request by jomit1612**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -191,3 +191,40 @@ describe("GET api/articles,", () => {
       });
   });
 });
+describe("GET api/articles/:article_id/comments", () => {
+  test("status(200) returns an array of comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
```
Comment (sirKevlar) [inline __tests__/app.test.js:199]: If using a forEach in the test suite like this, we need to check the array length to avoid a false positive

File: __tests__/app.test.js [hunk 2]
```diff
@@ -191,3 +191,40 @@ describe("GET api/articles,", () => {
...
+  });
+  test("status(404) returned bad request when passed an invalid id", () => {
+    return request(app)
+      .get("/api/articles/blue/comments")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("bad request");
+      });
+  });
+});
```
Comment (sirKevlar) [inline __tests__/app.test.js:238]: We could do with one more test her for an article which has no associated comments (status 200 returns empty array)
